### PR TITLE
feat: get Stripe keys from environment file

### PR DESCRIPTION
**Pull request by mitjans**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

By default, get Stripe keys from `.env` file. This allows us to set Stripe production keys in environment variables when running the application, while the `nuxt.config.ts` file will only contain test keys (`sk_test` and `pk_test`).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Comment by fuentesloic** [comment]
@mitjans can you update the readme as well please 🙏 thanks a lot! it's a great addition! 🚀

**Comment by mitjans** [comment]
Done! Let me know if you'd change anything :)

**Review by fuentesloic — APPROVED** [review]
CR done! LGTM! 🚀 will be deploy once you will have squashed and merged it 
Thanks again @mitjans 🙏

**Pull request merged**
